cityofaustin/joplin
# Issue 4641: add body field to official documents

https://github.com/cityofaustin/techstack/issues/4641

# Description

The OfficialDocumentPage now has a body field. It is now ready for us to do something with it.

# Types of changes



# How can this be tested?
https://joplin-pr-4641-body.herokuapp.com/admin/pages/31/edit/?next=/admin/pages/31/edit/#tab-content
Go here. See if you can add body text to the document. Nothing will show up on Janis yet, this is just the backend part.

# Checklist:
